src/api/institutionApi.ts

```diff
@@ -0,0 +1,86 @@
+import Axios from 'axios';
+import { API_URL } from '../utils/constants';
+import {
+  Institution,
+  InstitutionSubUnit,
+  InstitutionPresentationModel,
+  emptyInstitutionNames,
+} from '../types/institution.types';
+
+export enum InstituionApiPaths {
+  INSTITUTION = '/institution',
+  UNIT = '/institution/unit/',
+}
+
+export const queryInstitution = async (searchTerm: string) => {
+  try {
+    const response = await Axios({
+      method: 'GET',
+      url: `${API_URL}${InstituionApiPaths.INSTITUTION}?name=${searchTerm}`,
+    });
+    return response.data;
+  } catch (e) {
+    return [];
+  }
+};
+
+export const getInstitutionSubUnit = async (cristinUnitId: string) => {
+  try {
+    const response = await Axios({
+      method: 'GET',
+      url: `${API_URL}${InstituionApiPaths.UNIT}${cristinUnitId}`,
```

Comment:
Trenger ikke `${API_URL}` her siden vi har det på rota (i App)
`
// Set global config of axios requests
      Axios.defaults.baseURL = API_URL;
`

Reply:
samme overalt i denne filen